### PR TITLE
ADD: addin python parser for the retext-equality dataset

### PR DESCRIPTION
**Pull request by Guilhermeslucas**
This is the first part to address totally the issue #14 !

This adds the python parser for the [rules](https://github.com/retextjs/retext-equality/blob/main/rules.md) presented on retext, alongside with the table of the dataset and the processed output!

Let me know what you think!

### CODE REVIEW

**Comment by fernandoBRS** [comment]
@Guilhermeslucas  should we update the prerequisites section on README to add python 3.x?

**Comment by fernandoBRS** [comment]
Loved the structure of the txt file, great job! 👍

**Comment by Guilhermeslucas** [comment]
@fernandoBRS  I am really not sure if we should! I mean, Python3 is not a requirement to run the linter, but to run the parser! The parser is more for the developers of the project and not for the end user! What we can do is maybe add a developing dependencies or a separate group of dependencies, and then add Python3!

What do you say?

**Comment by fernandoBRS** [comment]
@Guilhermeslucas what if we create a `developers` section on README with the steps to run the parser?

**Comment by Guilhermeslucas** [comment]
Makes sense! Ill add it right now!

**Comment by Guilhermeslucas** [comment]
Done!

File: README.md [hunk 1]
```diff
@@ -36,6 +36,22 @@ To check the installed version, run the following command:
 inclusivelint -v
 ```
 
+## Development Requirements
+
+The parsers of this project are writtne using Python3. So, in order to update the dictionary that is located in
```
Comment (fernandoBRS) [inline README.md:41]: There is a typo on `writtne`

Comment (fernandoBRS) [reply README.md:41]: What if we rephrase it by `The linter leverages a terms parser written in Python 3. If there is a need to update the dictionary located on parsers/outputRelation.txt, make  sure you have Python 3.x installed then run it`?

File: README.md [hunk 2]
```diff
@@ -36,6 +36,22 @@ To check the installed version, run the following command:
...
 
+## Development Requirements
+
+The parsers of this project are writtne using Python3. So, in order to update the dictionary that is located in 
+```parsers/outputRelation.txt```, install Python3.* and run:
+
+```sh
+python retextEqualityParser.py
+```
+Note: be sure that ```python``` command is pointing to python3.*. To check it do:
```
Comment (fernandoBRS) [inline README.md:47]: What if we add this note step before the instruction to run the `retextEqualityParser.py`?

File: README.md [hunk 3]
```diff
@@ -36,6 +36,22 @@ To check the installed version, run the following command:
...
+``` sh
+python --version
+```
+
+The parser will get the ```wordsTable.md``` file in the same directory and output the dictionary to the
```
Comment (fernandoBRS) [inline README.md:52]: What if we rephrase it by something like `The parser receives the wordsTable.md as input and create/update the dictionary on the outputRelation.txt, located on the same directory.`?

File: README.md (hunk 2 above)
Comment (Guilhermeslucas) [reply README.md:47]: Sure! Done!

File: README.md (hunk 3 above)
Comment (Guilhermeslucas) [reply README.md:52]: Makes sense! Fixed as well!

File: README.md (hunk 1 above)
Comment (Guilhermeslucas) [reply README.md:41]: Makes sense! Fixed!

**Comment by Guilhermeslucas** [comment]
Fixed!

**Pull request merged**